tpgtools/utils/general.go

```diff
@@ -46,7 +42,7 @@ func formatComparator(formats []string) func(i, j int) bool {
 	}
 }
 
-func stringInSlice(a string, list []string) bool {
+func StringInSlice(a string, list []string) bool {
```

Comment:
Same as above- we'll want to comment this (and any other newly-exported fields)

Reply:
will add